src/Workspaces/Core/Portable/EmbeddedLanguages/Json/JsonOptions.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Options.Providers;
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.Json
+{
+    internal class JsonOptions
+    {
+        public static PerLanguageOption<bool> ColorizeJsonPatterns =
```

Comment:
Could you share screenshots?
Also, are we sure those settings should be per-language (C# vs. VB)?

Reply:
Screenshots of what?  Colored json?  If so, it's at the top of hte file.  And the settings should be per-language.  People should be allowed to say "i want to turn on embedded json in C# files, but not in VB files" (or vice versa).  

Options we expose are almost always per-language so that they can show up in tools|options and be independently flipped in the C# or VB sections.  We have some exceptiosn (like "full solution analysis"), but that's because we don't have a good system for even defining what it means for that setting to be single-language :)

Reply:
I meant the settings page  ;-)

Reply:
Gotcha.  Looks like this:

![image](https://user-images.githubusercontent.com/4564579/38476021-7ef54f52-3b60-11e8-8086-9f86a466aa4f.png)